src/utils/user.ts

```diff
@@ -0,0 +1,48 @@
+import cheerio, { CheerioAPI } from "cheerio";
+
+import axios from "axios";
+
+interface UserProfile extends CheerioAPI {
+  kind: "UserProfile";
+}
+
+export interface User {
+ name: string;
+ pseuds: string;
+ url: string;
+ id: string;
```

Comment:
Usually "id" should be first. IDK that it's a convention, but it's what my heart says.